README.md

```diff
@@ -128,13 +128,30 @@ go install github.com/charmbracelet/vhs@latest
 
 ## The VHS Server
 
-VHS has an SSH server built in! When you self host VHS you can access it as
+VHS has an SSH server built in! When you self-host VHS you can access it as
 though it were installed locally. VHS will have access to commands and
 applications on the host so you don't need to install them on your machine.
 
-Ensure you've set `~/.ssh/authorized_keys` with the public keys to which
-you want grant SSH access. You can change the file VHS looks at by setting
-`VHS_AUTHORIZED_KEYS_PATH` in your environment.
+<details>
+<summary>Server Configuration</summary>
+
+You can customize the VHS server by setting the following environment variables:
+
+| Environment                | Description                        | Default             |
+| -------------------------- | ---------------------------------- | ------------------- |
+| `VHS_PORT`                 | The port to listen on.             | `1976`              |
+| `VHS_HOST`                 | The host to listen on.             | `localhost`         |
+| `VHS_GID`                  | Group ID to run the server as.     | Current user's GID  |
+| `VHS_UID`                  | User ID to run the server as.      | Current user's UID  |
+| `VHS_KEY_PATH`             | The path to the SSH key to use     | `.ssh/vhs_ed25519`  |
+| `VHS_AUTHORIZED_KEYS_PATH` | The path to authorized keys files  | Empty               |
+
+> **Warning**:
+> An empty `VHS_AUTHORIZED_KEYS_PATH` means that your server is publically
+> accessible. Change this to `~/.ssh/authorized_keys` or another path to ensure
```

Comment:
I think this should be `.ssh/authorized_keys` instead to create the file in the binary's CWD

Reply:
it can be, but i'll not create the file if it doesn't exist...

I openend an issue on wish to work on auto-reloading the authorized keys file, maybe we could handle it there too